.travis.yml

```diff
@@ -46,3 +49,7 @@ script:
   - cd ..
   # Run a complete deduplication
   - (cd deduplicator && cargo run --release -- --osm ../importers/osm/test-files/relations_ways.pbf --output-csv test.csv)
+  - if [ "$TRAVIS_PULL_REQUEST_BRANCH" == "" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
+    docker build -t qwantresearch/addresses-importer .;
```

Comment:
Maybe we want to add some metadata to the image (from commit?) like we are doing here: https://github.com/QwantResearch/kartotherian_docker/blob/577c3172892b850310e9304a47add21a5aa043c7/.travis.yml#L20.

I'm also quite a beginner with docker and I'm not sure what it does, the documentation literally says nothing more interesting than:

> Set metadata for an image

Reply:
Yes that's a good idea!

Docker labels are basically "key=value" metadata that you can retrieve with `docker inspect`
https://docs.docker.com/config/labels-custom-metadata/